src/vdaf/prio3.rs

```diff
@@ -620,6 +621,18 @@ impl<const SEED_SIZE: usize> Encode for Prio3PublicShare<SEED_SIZE> {
     }
 }
 
+impl<const SEED_SIZE: usize> PartialEq for Prio3PublicShare<SEED_SIZE> {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
For consistency, note that you short-circuit on whether joint randomness is used or not.